store/sqlite/entries.go

```diff
@@ -32,7 +33,17 @@ func (s Store) GetEntriesMetadata() ([]picoshare.UploadMetadata, error) {
 				entries_data
 			GROUP BY
 				id
-		) sizes ON entries.id = sizes.id`)
+		) sizes ON entries.id = sizes.id
+	LEFT OUTER JOIN
+		(
+			SELECT
+                entry_id,
```

Comment:
Can we use tabs here so that it's aligned with the next line?

Reply:
Thanks, I fixed it